content/docs/config.md

```diff
@@ -1,4 +1,4 @@
-# Using CML
+# CML Configuration
```

Comment:
that's a big rename. What's the reason? Would have to update wording & explanation in other places; e.g. the overview in https://cml.dev/doc

Reply:
Yes TBH I'm not 100% sure of this. I see that the original intention of the page is more "Usage" than config but the contents overlap with other docs so this is what I came up with: move the other usage info to existing docs and just leave the config part here.

Relates to the [question](https://github.com/iterative/cml.dev/pull/382#pullrequestreview-1185693709) on the **CML Reports** section.

Reply:
While I agree "Usage" might not be accurate (maybe rename to "User Guide: Introduction"?), "Config" is definitely not right. It's not the intention, and also I don't think it's accurate. My reasoning is that `.gitlab-ci.yml`, `.github/workflows/cml.yaml`, `bitbucket-pipelines.yml` are NOT "CML config files" (it's a bit like calling `Dockerfile` a "DVC config file" just because it contains `RUN dvc repro`).


For clarification, my vision is:

- we need 4 major sections (the 4 cards in https://cml.dev/doc)
  1. beginner intro to reporting ("Get Started")
  2. advanced/in-depth guide to reporting ("Usage", happy to rename to "User Guide" thus matching https://dvc.org/doc)
  3. specific use case/integration guides (atm only "DVC with CML" but in future will include #196, #207, etc.)
  4. guide to `runner` ("Self-hosted Runners")
- the description of said sections (the text in the 4 cards in https://cml.dev/doc) is accurate, but the titles (e.g. "Usage") and the page contents might need tweaking

WDYT?

Reply:
> maybe rename to "User Guide: Introduction"

👍🏼 

> .gitlab-ci.yml, .github/workflows/cml.yaml, bitbucket-pipelines.yml are NOT "CML config files"

Good point, these are CI config files so we're not really configuring CML there. BTW is there any CML configuration?

> we need 4 major sections (the 4 cards in https://cml.dev/doc

I feel like **CML with DVC** is part of the user guide. That said, 3 cards would look strange (design issue though).

> WDYT?

In general agree! (Refer to [framework](https://www.notion.so/iterative/Framework-7e86e484e50e42529df3dbfe32b4e987)). Will update PR accordingly ⏳ 

> iii. specific use case/integration guides

Goes back to our discussion on what is a use case (somewhere) 😋 Summary: CML has one main high-level use case (it's in the name: CI/CD for ML), so we may not need that section in docs. There's https://cml.dev/#use-cases though... (again, separate discussion on the terminology)

> i. beginner intro to reporting ("Get Started")

p.s. Let's discuss the details of what and how to cover in the GS separately

Reply:
> Will update PR accordingly ⏳

Done. PTAL